git-hub

```diff
@@ -246,9 +317,9 @@ def find_match(open_or_closed, list_of_dictionaries, keyword, user, comment, num
                 if label.upper() in temp_label.upper() and comment.upper() in temp_comment.upper() and user.upper() in temp_user.upper() and branch.upper() in temp_branch.upper():
                     appeared_before = True
                     if(open_or_closed):
-                        all_prs[sub_dictionary['modified']] = '{0} \x1b[0;32;40m {1} \x1b[0m {2}/{3} {4}'.format(sub_keys, "O", sub_dictionary['user'], sub_dictionary['branch'], sub_dictionary['comment'])
+                        all_prs[sub_dictionary['modified']] = '{0} {1} {2} {3} {4}/{5} {6}'.format(sub_keys, ansi.GREEN, "O", ansi.CLOSE, sub_dictionary['user'], sub_dictionary['branch'], sub_dictionary['comment'])
                     else:
-                        all_prs[sub_dictionary['modified']] = '{0} \x1b[0;31;40m {1} \x1b[0m {2}/{3} {4}'.format(sub_keys, "C", sub_dictionary['user'], sub_dictionary['branch'], sub_dictionary['comment'])
+                        all_prs[sub_dictionary['modified']] = '{0} {1} {2} {3} {4}/{5} {6}'.format(sub_keys, ansi.RED, "C", ansi.CLOSE, sub_dictionary['user'], sub_dictionary['branch'], sub_dictionary['comment'])
```

Comment:
Use f-strings for all these; it will make them a lot easier to read.  Also note that code should not be longer than 80 chars per line.